swingset/src/main/java/com/nqadmin/swingset/utils/SSFormViewScreenHelper.java

```diff
@@ -873,12 +881,20 @@ protected void updateScreen() {
 
 	} 
 
-
 	/**
-	 * Used to update any SSDBComboBox that change when parent ID changes (e.g. a
-	 * record selector combo). This is called from initScreen() after bindComponents()
-	 * and addComponents(). It is also called from updateScreen() after the rowset
-	 * has been requeried.
+	 * Used to update any SSDBComboBox that change when parent ID changes (i.e.,
+	 * whenever the rowset for the screen changes). This is called from initScreen()
+	 * after bindComponents() and addComponents(). It is also called from
+	 * updateScreen() after the rowset has been requeried.
+	 * <p>
+	 * Generally there will be calls to cmbMySSDBCombo.setQuery(myNewCmbSQL); and
+	 * cmbMySSDBCombo.execute();
+	 * <p>
+	 * If the SSDBComboBox query is unchanged regardless of the rowset, then it is
+	 * sufficient to just call cmbMySSDBCombo.execute();
+	 * <p>
+	 * It is REDUNDANT/UNNECESSARY to call cmbMySSDBCombo.execute() here and in
+	 * bindComponents().
 	 */
 	protected abstract void updateSSDBComboBoxes();
 
```

Comment:
First time I've looked into a screen help file. Looks like this situation was already covered, at least partly :-)